packages/charting/src/main.jsx

```diff
@@ -5,39 +5,35 @@ import { Collapsible } from '@pie-lib/render-ui';
 import { Chart, chartTypes } from '@pie-lib/charting';
 import isEqual from 'lodash/isEqual';
 
-
 export class Main extends React.Component {
   static propTypes = {
     classes: PropTypes.object,
     model: PropTypes.object.isRequired,
     onAnswersChange: PropTypes.func,
+    categories: PropTypes.array
   };
 
   static defaultProps = { classes: {} };
 
-  constructor(props) {
-    super(props);
-    const { model: { data = [] } } = props;
+  state = {};
 
-    this.state = { data };
+  componentDidMount() {
+    this.props.onAnswersChange(this.props.model.data);
```

Comment:
i'm confused - why would you trigger this from something in the model? i thought the session value was passed in to `categories`?